.github/workflows/linux-ci.yml

```diff
@@ -35,22 +35,17 @@ jobs:
         CC: /usr/bin/clang
         CXX: /usr/bin/clang++
     - name: CMake (coverage/clang-tidy/clang-asan)
-      if: github.ref == 'refs/heads/master'
       run: |
-        cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Debug -DTW_CODE_COVERAGE=ON -DTW_ENABLE_CLANG_TIDY=ON -DTW_CLANG_ASAN=ON
-      env:
-        CC: /usr/bin/clang
-        CXX: /usr/bin/clang++
-    - name: CMake (coverage)
-      if: github.ref != 'refs/heads/master'
-      run: |
-        cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Debug -DTW_CODE_COVERAGE=ON
+        cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Debug -DTW_UNITY_BUILD=ON -DTW_CODE_COVERAGE=ON -DTW_ENABLE_CLANG_TIDY=ON -DTW_CLANG_ASAN=ON -GNinja
       env:
         CC: /usr/bin/clang
         CXX: /usr/bin/clang++
     - name: Build and test
       run: |
-        make -Cbuild -j12 tests TrezorCryptoTests
+        cd build
+        ninja tests 
```

Comment:
can we run `make ninja` or somehow?

Reply:
No ninja is the build system we use, multiple generator are not possible without cmake presets